SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj

```diff
@@ -176,7 +181,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos";
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = SWIFT_PACKAGE;
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = Default;
+				SWIFT_VERSION = 4.0;
```

Comment:
A swift 4 framework and swift 3.2 framework should work side-by-side. At least that was what apple said in their videos :) But I could change it to 3.2 for now and you could upgrade to 4.0 later!